README.md

```diff
@@ -53,6 +53,9 @@ dynamic-theme-fixes.config
 
 example.com
 
+IGNORE INLINE STYLE
```

Comment:
Since it's rarely used, let's put it on the last place.

Reply:
That's now how our code work IGNORE INLINE STYLE I first one

Reply:
Then I think the code should be fixed.

Reply:
Okay